build-automation.js

```diff
@@ -0,0 +1,137 @@
+const util = require("util")
+
+const exec = util.promisify(require("child_process").exec);
+
+const https = require("https");
+
+const mapSeries = (arr) => {
```

Comment:
what does this do that `Promise.all` doesn't?

Reply:
I wanted to run the update.sh script more than once, if needed, but sequentially. I don't know if that was an actual requirement, but my first thought was that I wasn't sure if the more than update.sh script running in parallel would create conflicts or mess up with some files.
If the sequential part of those promises isn't really needed (eg it won't break anything to run more than one update.sh script instance in parallel), I'm happy to use Promise.all.

@SimenB

Reply:
If you wanna run sequentially, just do


```js
for (const thing of things) {
  await thing.something();
}
```

Reply:
I really overcomplicated this one. And I don't even know why.
Thank you. I will make the change.